app/services/pagelets/pagelet.rb

```diff
@@ -0,0 +1,20 @@
+module Pagelets
+  class Pagelet
+    attr_reader :name, :partial, :priority, :opts
+
+    def initialize(name, partial, priority, opts)
+      @name = name
+      @partial = partial
+      @priority = priority
+      @opts = opts
+    end
+
+    def <=>(other)
+      priority <=> other.priority
+    end
+
+    def snake_name
+      @name.gsub(/\s+/, "_").underscore
+    end
+  end
+end
```

Comment:
Mind adding `method_missing` returning an option from the `opt` hash? So instead of

    pagelet.opts[:probe_name]

I can write

    pagelet.probe_name